Essentials/src/main/java/com/earth2me/essentials/config/processors/DeleteIfIncompleteProcessor.java

```diff
@@ -0,0 +1,41 @@
+package com.earth2me.essentials.config.processors;
+
+import org.spongepowered.configurate.ConfigurationNode;
+import org.spongepowered.configurate.objectmapping.meta.Processor;
+import org.spongepowered.configurate.serialize.SerializationException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DeleteIfIncompleteProcessor implements Processor<Object> {
+    @Override
+    public void process(Object value, ConfigurationNode destination) {
+        if (value == null) {
+            return;
+        }
+
+        try {
+            if (value instanceof IncompleteEntity && ((IncompleteEntity) value).isIncomplete()) {
+                destination.set(null);
+            } else if (value instanceof List<?>) {
```

Comment:
Could maybe check other collection types? Although not a priority for fixing this bug

Reply:
other collections are going to make me cry